docs/site/tutorials/protocol_oriented_generics.ipynb

```diff
@@ -0,0 +1,1141 @@
+{
```

Comment:
<p>Similarly to the previous section, I think that the potential controversy of comparing Swift with everyone's favorite languages can be avoided by saying something like "In addition to classes which have reference semantics, Swift supports enums and structs that are passed by value. Enums and structs support many features provided by classes, albeit with a few restrictions. Let's take a look".</p><p><br></p><p>I believe that Swift's features are deep enough and useful enough to stand on their own, without necessarily referring to alternatives which are supposedly doing worse in some regards. The latter may make the message more powerful, but it also invites further questions and imposes an additional burden of proof, which both make the task of writing more complicated.</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/swift/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>